atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -212,12 +209,12 @@ public void successfulRequestCausesHostToBeRemovedFromBlacklist() {
 
         assertThatThrownBy(() -> runNoopWithRetryOnHost(HOST_1, cassandraClientPool))
                 .isInstanceOf(AtlasDbDependencyException.class);
-        assertThat(blacklist.contains(HOST_1), is(true));
+        assertThat(blacklist.contains(HOST_1)).isEqualTo(true);
```

Comment:
These will be automatically cleaned up into `isTrue()` next time the robot runs.